draft-ietf-quic-transport.md

```diff
@@ -124,9 +124,9 @@ provides:
 
 * Authenticated and encrypted header and payload
 
-QUIC uses UDP as a substrate to avoid requiring changes in legacy client
+QUIC uses UDP as a substrate to avoid requiring changes to legacy client
 operating systems and middleboxes.  QUIC authenticates all of its headers and
-encrypts most of the data it exchanges, including its signaling, to avoid
+encrypts most of the data it exchanges -- including its signaling -- to avoid
```

Comment:
I hate em--dashes done this way.

Reply:
Do you mean you don't want to use em-dashes here or that you would like me to use some other typographical convention than --

Reply:
We're using hyphens instead of em-dashes... I used to write em-dashes, but @martinthomson normalized it all to hyphens, so he'll have to explain :-)

That said, I think commas seem more appropriate here. Ultimately, I suspect the RFC-Editor will want to say something here.

Reply:
This is just what I learned from Professor Schulzrinne.  If we had real em-dashes, I'd use them, but the budget version is just bad.